packages/mock/src/mocks/structuredefinitions.json

```diff
@@ -14812,7 +14812,12 @@
           "id": "Project",
           "path": "Project",
           "min": 0,
-          "max": "*"
+          "max": "*",
+          "base": {
```

Comment:
Hm, not sure why all of these `base` values started popping up...

`base` should not be required, because in the vast majority of cases it is duplicate data (i.e., our schema parsing code should be able to fall back to the `min`, `max`, and `path` values if `base` is missing).

I haven't looked into this code in a while, it's probably a small unrelated change somewhere else...  Do you mind taking a quick look?  If you don't see anything obvious, (1) @mattwiller might now, because I think he was working on some code changes with `base` before, or (2) we can accept inflated storybook size and ship as-is.

Reply:
It might not even be that inflated, gzip will probably compress most of it away.

Reply:
Done 👍 -4500 lines
https://github.com/medplum/medplum/pull/4256/files#diff-1ecb3d5a1ea829cea8bfd6ac302cdcf2e1df7281543996f4244949823d8be60a